lib/pleaserun/detector.rb

```diff
@@ -25,14 +25,19 @@ def detect_platform
 
   def detect_systemd
     # Expect a certain directory
-    return false unless File.directory?("/lib/systemd/system")
+    return false unless File.directory?("/lib/systemd/system") || File.directory?("/usr/lib/systemd/system")
```

Comment:
16.04 does have `/usr/lib/systemd`:
```
root@ubuntu-xenial:~# ls /usr/lib/systemd/
boot  catalog  network  user  user-generators
```
That's why the change is now going to just use `/usr/lib/systemd`.  Each of the systems had that directory, even if it wasn't where the unit files went.